### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by SP4R0W**

### CODE REVIEW

File: blueprints/payload/payload.gd [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+	TIMER = $Timer
+	TIMER.wait_time = payload_range
+	TIMER.start()
+
+func _physics_process(delta: float) -> void:
+	global_position += direction.rotated(rotation) * speed * delta
+
+func _on_timer_timeout() -> void:
+	queue_free()
```
Comment (VargaDot) [inline blueprints/payload/payload.gd:42]: Old script file
```suggestion
```

File: blueprints/payload/payload.tscn [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+[sub_resource type="RectangleShape2D" id="RectangleShape2D_tkips"]
+
+[node name="Payload" type="Area2D"]
+script = ExtResource("1_aaav6")
+
+[node name="CollisionShape2D" type="CollisionShape2D" parent="."]
+shape = SubResource("RectangleShape2D_tkips")
+
+[node name="Timer" type="Timer" parent="."]
```
Comment (VargaDot) [inline blueprints/payload/payload.tscn:13]: Old scene
```suggestion
```

File: test_scene/test.tscn [hunk 3]
```diff
@@ -1,9 +1,16 @@
-[gd_scene load_steps=3 format=3 uid="uid://cjuweh32idvkx"]
+[gd_scene load_steps=5 format=3 uid="uid://cjuweh32idvkx"]
 
 [ext_resource type="Script" path="res://test_scene/test.gd" id="1_jlb5m"]
 [ext_resource type="PackedScene" uid="uid://cdg3qi6f4n7vf" path="res://weapons/pistol/pistol.tscn" id="2_tf4ak"]
+[ext_resource type="PackedScene" uid="uid://bn2f2gkcynlw3" path="res://blueprints/field/field.tscn" id="3_7byuc"]
+[ext_resource type="PackedScene" uid="uid://cp6ask5xmce30" path="res://blueprints/projectile/projectile.tscn" id="4_4nfha"]
```
Comment (VargaDot) [inline test_scene/test.tscn:6]: please use pistol_bullet scene, its in the pistol folder

File: test_scene/test.tscn [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
 
 [node name="Test" type="Node2D"]
 script = ExtResource("1_jlb5m")
 
 [node name="Pistol" parent="." instance=ExtResource("2_tf4ak")]
+reload_time = 1.0
+reload_time_empty = 1.5
```
Comment (VargaDot) [inline test_scene/test.tscn:13]: this adds 1.5 on top of the reload_time, im just saying

**Review by VargaDot — CHANGES_REQUESTED** [review]

File: test_scene/test.tscn (hunk 3 above)
Comment (SP4R0W) [reply test_scene/test.tscn:6]: thats already done

**Review by VargaDot — APPROVED** [review]

**Comment by VargaDot** [comment]
great work @SP4R0W

**Pull request merged**